packages/fx-core/src/common/projectTypeChecker.ts

```diff
@@ -34,6 +35,7 @@ export interface ProjectTypeResult {
   manifestVersion?: string;
   dependsOnTeamsJs?: boolean;
   isSPFx?: boolean;
+  officeProjectType?: string;
```

Comment:
Better to use `officeAddinProjectType` is more precise, because there are another concept that M365 app can run in office platform, which will cause confusion with the concept of Office Add-in.

Reply:
Modified.